### PR TITLE
feat: httpc_params_loader (httpc default timeout refactor)

### PR DESCRIPTION
**Pull request by peppelinux**
A refactor of the previous commit
https://github.com/IdentityPython/JWTConnect-Python-CryptoJWT/pull/94#pullrequestreview-775725040

### CODE REVIEW

File: src/cryptojwt/key_bundle.py [hunk 1]
```diff
@@ -168,6 +168,13 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+def httpc_params_loader(httpc_params):
```
Comment (jschlyter) [inline src/cryptojwt/key_bundle.py]: Perhaps put this in `utils.py`?

**Review by jschlyter — DISMISSED** [review]

File: src/cryptojwt/key_bundle.py (hunk 1 above)
Comment (peppelinux) [reply src/cryptojwt/key_bundle.py]: sounds good, ok, I'm doing it right now

Comment (peppelinux) [reply src/cryptojwt/key_bundle.py]: done
https://github.com/IdentityPython/JWTConnect-Python-CryptoJWT/commit/703b0d861eff4ed2bfc6efc5a1c9b310ad81d1cf

**Review by jschlyter — DISMISSED** [review]

**Comment by peppelinux** [comment]
fixed all the warnings that raises up during unit tests execution
https://github.com/IdentityPython/JWTConnect-Python-CryptoJWT/pull/95/commits/a60c1d4b0f2fba24912f6e22a83adeb9b33173a1

something went wrong but only in gh actions CI not in my workstation, I'm on it.

**Comment by peppelinux** [comment]
fixed, all done @rohe @jschlyter

**Review by rohe — APPROVED** [review]

**Review by jschlyter — APPROVED** [review]

**Pull request merged**